steamworks/interfaces/utils.py

```
@@ -0,0 +1,160 @@
+
+    def SetOverlayNotificationInset(self, horizontal: int, vertical: int) -> None:
+        """Sets the inset of the overlay notification from the corner specified by SetOverlayNotificationPosition
+
+        :param horizontal: int
+        :param vertical: int
+        :return: None
+        """
+        self.steam.SetOverlayNotificationInset(horizontal, vertical)
+
+
+    def SetOverlayNotificationPosition(self, position: ENotificationPosition) -> None:
```

Ruff (0.14.5)

`ENotificationPosition` may be undefined, or defined from star imports

(F405)